### ISSUE TITLE
Prepare Exchange

### ISSUE THREAD

**Issue by wilsond3010** [MicrosoftDocs/Azure-RMSDocs#615]
Wondering is this correct or should one be internal and external
Set-IRMConfiguration -internallicensingenabled $false
Set-IRMConfiguration -internallicensingenabled $true 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: db563c7f-28b3-abec-2310-b9c7d963b4e9
* Version Independent ID: 448c88a4-3e53-a50a-16ac-9b7e71faecab
* Content: [Migrate AD RMS-Azure Information Protection - Phase 1](https://docs.microsoft.com/en-us/azure/information-protection/migrate-from-ad-rms-phase1)
* Content Source: [Azure-RMSDocs/migrate-from-ad-rms-phase1.md](https://github.com/Microsoft/Azure-RMSDocs/blob/master/Azure-RMSDocs/migrate-from-ad-rms-phase1.md)
* Service: **information-protection**
* GitHub Login: @cabailey
* Microsoft Alias: **cabailey**

**Comment by cabailey** [comment]
@wilsond3010 No, it's correct as-is. When you disable and re-enable this parameter, Exchange picks up the new value you've just added for your tenant in the licensing location list.

**Comment by wilsond3010** [comment]
Thanks that was what I was wondering 


Sent from Yahoo Mail for iPhone


On Friday, September 14, 2018, 12:02 PM, Carol Bailey <notifications@github.com> wrote:


@wilsond3010 No, it's correct as-is. When you disable and re-enable this parameter, Exchange picks up the new value you've just added for your tenant in the licensing location list.

**Comment by cabailey** [comment]
#please-close
